drivers/wireless/ieee80211/bcm43xxx/bcmf_sdpcm.c

```diff
@@ -317,7 +316,7 @@ int bcmf_sdpcm_readframe(FAR struct bcmf_dev_s *priv)
             DEBUGPANIC();
           }
 
-        bcmf_dqueue_push(&sbus->rx_queue, &sframe->list_entry);
+        list_add_head(&sbus->rx_queue, &sframe->list_entry);
```

Comment:
should we add tail?

Reply:
I just keep the original style,  either add tail or head is fine.